packages/blitz-rpc/src/loader-utils.ts

```diff
@@ -69,6 +69,15 @@ export function convertPageFilePathToRoutePath({
     path = path.replace(normalize(appRoot), "")
     for (const extraPath of extraRpcBasePaths) {
       path = path.replace(join(normalize(appRoot), extraPath.replace("/", sep)), "")
+
+      // change paths included in includeRPCfolders setting only
+      if (
+        normalize(absoluteFilePath).includes(
+          normalize(extraPath.replace("../", "").replace("../", sep)),
+        )
+      ) {
+        path = join(normalize(extraPath.replace("../", "").replace("../", sep)), path)
+      }
```

Comment:
can you explain how this fixes it? 

We should try to find another way to fix this that doesn't rely on hard coding `../` values.

lastly, can you update the integration test with a reproduction of the bug you have in your app? I need that to understand what the problem is and so we can make sure it doesn't break in the future.

Reply:
> We should try to find another way to fix this that doesn't rely on hard coding ../ values.

Yes, this is what I mentioned in discord about needing some help.  I tried some functions like normalise and I couldn't find a better way.  I absolutely agree with your comment.

> can you explain how this fixes it?

Primarily, I had to update resolverFilePath in other files to reflect the original change from relative to absolute path in the original PR.  Doing this change, ensured that the output was always uniform.

Secondly, if our next config, blitz param is like this:

```
resolverPath: "root",
includeRPCFolders: ["../../modules"],
```

The request urls become:

> http://localhost:3001/api/rpc/modules/dam/queries/getDam

Otherwise if I remove that code.  It becomes:

> http://localhost:3001/api/rpc/dam/queries/getDam

I think this approach is correct, as if there was another workspace with duplicate names for the components and or queries/mutations.  This may lead to issues.  I think we should cover all use cases.

Reply:
Would this be a better change?

```
// change paths included in includeRPCfolders setting only
const baseRPCFolder = sep + extraPath.replace(/^(?:\.\.\/)+/, "")

if (normalize(absoluteFilePath).includes(baseRPCFolder)) {
    path = join(baseRPCFolder, path)
}
```

Reply:
@paulm17 get the integration test added, and then I look at how to refactor this